docs/vision-usage.rst

```diff
@@ -31,8 +31,38 @@ or pass in ``credentials`` and ``project`` explicitly.
      >>> client = vision.Client(project='my-project', credentials=creds)
 
 
+Creating an :class:`~google.cloud.vision.image.Image` object.
+-------------------------------------------------------------
+
+The :class:`~google.cloud.vision.image.Image` object is used to load your image
+data from sources such as a Google Cloud Storage URI, raw bytes, or a file.
+
+
+**From a Google Cloud Storage URI**
```

Comment:
Sure, I can use `~~~~~~` but that looks really close in size to `-------` when rendered.